action.yml

```diff
@@ -199,6 +199,26 @@ runs:
           cd /tmp/skaffold
           skaffold config set collect-metrics false
           skaffold deploy
+
+          if [ $? -eq 0 ]; then
+            # Publish Artifact Info on success
+            response=$(curl -i -X 'POST' "$CLOUDBEES_API_URL/v2/workflows/runs/artifactinfos" \
+              -d "{
+                \"runId\": \"$CLOUDBEES_RUN_ID\",
+                \"runAttempt\": \"$CLOUDBEES_RUN_ATTEMPT\",
+                \"name\": \"$RELEASE_NAME\",
+                \"url\": \"$CHART_LOCATION\",
```

Comment:
For the artifact URL, whenever possible we need to use a url that will match the url from the corresponding publish action. In case of `helm-install` it is `helm-push`. So when the chart url is an An [OCI-based](https://helm.sh/docs/topics/registries/) chart URL, then specify the artifact url in purl format. We are going to do the same in `helm-push` - https://docs.google.com/spreadsheets/d/1vuzHtpyeupI9rfgHXP2tls6ZfuM6qeIIGH81ZTux1a0/edit?pli=1&gid=0#gid=0.